sentinelsat/scripts/cli.py

```diff
@@ -255,6 +261,14 @@ def cli(
         with open(os.path.join(path, "search_footprints.geojson"), "w") as outfile:
             outfile.write(gj.dumps(footprints_geojson))
 
+    if quicklook:
+        failed_quicklooks = api.download_quicklooks(products, path)
+        if failed_quicklooks:
```

Comment:
Please do not write a text file with failed quicklooks. Just write a message to the terminal or so. Fine for the function to return this information, though. Maybe something like `f"Some quicklooks failed to download: {len(products)} out of {len(failed_quicklooks)}."`. If someone needs logic about what to do in case of failing quicklooks, they need to use the Python API.

Reply:
Thanks, will change this and log to screen instead. Had a look at writing tests over the weekend which made me rewrite so the functions take parameters similar to download(), download_all() and then can test them similar to what you are already doing there.

Will try and get that commit up tomorrow, appreciate you guys having a look at this.